src/utils.rs

```diff
@@ -32,7 +32,7 @@ where
     let source_path = source_path.as_ref();
     let file = fs::File::create(zip_file.as_ref())?;
     let mut zip = zip::ZipWriter::new(file);
-    let options =
+    let options: zip::write::FileOptions<zip::write::ExtendedFileOptions> =
```

Comment:
essayes avec 
```suggestion
    let options =
        zip::write::SimpleFileOptions::default().compression_method(zip::CompressionMethod::Deflated);
```

tu n'es pas obligé de clone après

Reply:
Done in https://github.com/hove-io/transit_model/pull/952/commits/5bac502aa4f766202da882b22fda781c37479d86
There are already tests on zip compression. Results are ok.